src/models/sunburst.js

```diff
@@ -68,8 +70,8 @@ nv.models.sunburst = function() {
             y.range([0, radius]);
 
             node = node || data;
-            path =
-                g.data(partition.nodes).enter()
+            partition.value(modes[mode]);
```

Comment:
You might want to add a default in there if they try to set mode to something crazy... so like:

    partition.vaoue(modes[mode] || modes['count']);